### ISSUE TITLE
Spending ringct to ct ends up stuck in mempool due to wrong fee estimation.

### ISSUE THREAD

**Issue by DrBrainstorm** [Veil-Project/veil#296]
Spending zerocoin to self ends up in split balance between ring_ct and ct. In an attempt to spend ring_ct to ct txn does not get broadcasted due to what seem a wrong fee estimation. Including below part of debug.log
2019-01-15T14:43:14Z Fee Calculation: Fee:617500 Bytes:6174 Needed:617400 Tgt:0 (requested 0) Reason:"PayTxFee set" Decay 0.00000: Estimation: (-1 - -1) nan% 0.0/(0.0 0 mem 0.0 out) Fail: (-1 - -1) nan% 0.0/(0.0 0 mem 0.0 out)
2019-01-15T14:43:14Z CommitTransaction:
CTransaction(hash=79f1546b8a, ver=2, vin.size=1, nLockTime=0)
    CTxIn(COutPoint(0000000000, 4294967200), scriptSig=)
    CScriptWitness(22c5012ad4013041af018201027b5b, 8de6bcbf14e3ea23c478252248452e4132ceb9712d0852cbbd43beb525848a8eab7918e9ac15eea091ef26e19403ea53066c4bf556ec82fb163ae4f2bcbbd28bc7c2b9680d70a8378d40db31463d18c6110e9e989394a5da99be914bef4c26330f77b6d731cf7a947bfa82d60d3b143b93a5902d3b23c2fa6b2011981ef92f334a007faf01aeee2770f6f2f5a7eed770afd5ed9bf334d0de038e54ebcb50cf297d34fa3298bb6d3ba567f84e2bfeccd900db6b13c27a342205d18220b4404bf9bbfc6b78feac1cc2b32c10fdb499019036d5173aa10affb24a819234e069e29f11a93f9d572f89487d74577c883ad50d9ec59e217da334d4546b19cd9e7bd123c32307663d48000cd11d826d9ca00e930c55c93d66187b46ba727c3d4844f95c3055ab5a75dd325d2c463187b402940020b5f476b44f6677e2373c4b0ecf2503321c97fcb7df24e0ee6d5d7a13850cd2ade5f69266294b83684ca6e2d02fe6a7cc0ab25f93864ab0d59f3f85401b28c1b204a2f223c0344401ac8b9aed184478c1b886d0e9099a69c04fa576c7d7b50b8cd9d6a4e860e2beeccd6fae5de4d7cfe005aaad39747b0571c516f2e6158ee3805d35ce99c6a786dbe9de62c2321c7fbe683410f5ff6936d7c68b00472be61e084f5a6fc885ab96e4a912ff417bb53da09fb2bda2a6b67e178badab30e2efc15f708278469af9f5efa058752a87447c0f8834b16c4e9728d7e98933c7318d74fbf24b2c101e56ac611007a629e5e8322e1c6f0e48b6df24e7758aa001b3459ea789e5f1c9c686318bec51d845d7eed16a158c080924b0644523146d1fa8148b399e21002842ad91656057aa091a8b0296384c2aac1b210282de1d8e3b7bd06d1bf648c7262cb20b2ecd50f7e4a8d225c717dd137c61bbf70ff293b2c24f310bb257e4ea67cc566d65d51ad50cafa53198d17fb7e70e826497b19e4f021dda8a6c4932e819d4083f9cc7e15023eb2f8665440582a63aa761ffe9e83ae6be0c51cb8a0965cd34c6aec86198ad5a8bbcad)
    CTxOutData(data=069cd825)
    CTxOutCT(data=033bba648f9666eb3ff9bd3c5f87b3, scriptPubKey=76a914be2bde1a018f64543a319ac6)
    CTxOutCT(data=03fe689acdee926b5c7e5534bd2cad, scriptPubKey=76a91482e30f6983e4c97246ac6241)
2019-01-15T14:43:14Z [default wallet] AddToWallet 79f1546b8acb74f86b0b0a9249f0bf5e596814af70ec79282a4576640005400a  new
2019-01-15T14:43:14Z ERROR: AcceptToMemoryPoolWorker: Consensus::CheckTxInputs: 79f1546b8acb74f86b0b0a9249f0bf5e596814af70ec79282a4576640005400a, bad-commitment-sum (code 16)
2019-01-15T14:43:14Z CommitTransaction(): Transaction cannot be broadcast immediately, bad-commitment-sum (code 16)

v1.0.0.10-37615030e
Windows x64

**Comment by seanPhill** [comment]
I suppose that you have since used ```abandontransaction ...``` to clear that one. I had to do the same when I accidentally clicked the mint button twice and it generated a duplicate minting. I don't know how it managed to do it, as there were only enough in RingCT for that amount. Anyway, I momentarily saw a pop up message about unable to broadcast due to a ridiculously or maybe absurdly high fee. Eventually I abandoned the first minting also, after it remained unconfirmed for a long time, and the RingCT was still in its place. There was a debug.log message mentioning ```Reason:"MaxTxFee limit"```
